tests/python/contrib/test_dnnl.py

```diff
@@ -94,6 +94,9 @@ def partition_for_dnnl(mod, params=None, alter_layout=True, prune_subgraphs=True
             )
             with tvm.transform.PassContext(opt_level=3):
                 mod = seq(mod)
+
+    mod = dnnl.rewrite_dense_bias_gelu_reshape_last(mod)
```

Comment:
Could you please explain, why do we need this pass before "AlterOp" transformation passes?

Reply:
Hi Peskov, This pass _rewrite_dense_bias_gelu_reshape_last_ only matches the pattern "dense_bias_activation". If we are desired to put it after "AlterOp" pass, then I need to extend the capability of pass _rewrite_dense_bias_gelu_reshape_ .